<?xml version="1.0" ?>
<launch>

  <arg name="tool_name"		default="screw_tool_m3"/>

  <node name="$(arg tool_name)_test"
	pkg="aist_fastening_tools" type="suction_tool_test.py"
	output="screen">
    <param name="controller_ns"	value="$(arg tool_name)_suction_controller"/>
  </node>

  <include file="$(dirname)/suction_tool_controllers.launch">
    <arg name="tool_names"	value="$(arg tool_name)"/>
  </include>

</launch>
